specification/playwrighttesting/resource-manager/readme.md

```diff
@@ -29,7 +29,26 @@ title: PlaywrightTestingClient
 description: Azure Playwright testing management service
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-10-01-preview
+tag: package-2024-02-01
+```
+
+### Tag: package-2024-02-01
+
+These settings apply only when `--tag=package-2024-02-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-02-01'
+input-file:
+  - Microsoft.AzurePlaywrightService/stable/2024-02-01/playwrighttesting.json
+suppressions:
+  - code: ResourceNameRestriction
+    from: playwrighttesting.json
+    reason: We have enums defined for quota resource names which doesn't support string pattern validation.
+  - code: ProvisioningStateMustBeReadOnly
+    from: playwrighttesting.json
```

Comment:
Add readOnly true inside property definition, not where it is referenced. That will fix the linter issue and you can remove suppression for this.

Reply:
@visingla-ms the swagger json is being generated from TypeSpec file. We've followed the syntax to put visibility there. Changing it's place gives this error -

![image](https://github.com/Azure/azure-rest-api-specs/assets/71822730/f109baac-5233-403a-9dac-bc261cccbe25)